sci-mathematics/cgal/cgal-5.1.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake flag-o-matic
+CMAKE_BUILD_TYPE=Release
+
+MY_P=CGAL-${PV}
+
+DESCRIPTION="C++ library for geometric algorithms and data structures"
+HOMEPAGE="https://www.cgal.org/"
+SRC_URI="
+	https://github.com/CGAL/cgal/releases/download/v${PV}/${MY_P}.tar.xz
+	doc? ( https://github.com/CGAL/cgal/releases/download/v${PV}/${MY_P}-doc_html.tar.xz )"
+
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="LGPL-3 GPL-3 Boost-1.0"
+SLOT="0/14"
+KEYWORDS="~amd64 ~arm64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="doc examples +gmp mpfi ntl qt5"
+
+RDEPEND="
+	dev-cpp/eigen
+	dev-libs/boost:=[threads]
+	dev-libs/mpfr:0=
+	sys-libs/zlib:=
```

Comment:
zlib should be without :=

https://wiki.gentoo.org/wiki/Proxied_Maintainer_FAQ#Using_a_sub-slot_binder.2C_:.3D

Reply:
removed
done